ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset

```diff
@@ -0,0 +1,26 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: d8cfffae36be18b40b59e8e84d47a6ad, type: 3}
+  m_Name: UIUXMichaelsoft
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: d596ba1022aed3641984cad46486500b, type: 2}
+  positionName: UI/UX Designer
+  jobSummaryLines:
+  - We are looking for an individual with experience in UI/UX design.
+  - 'Requirements and qualifications:'
+  jobReqLines:
+  - '>Knowledge of equitable and usable UI design'
```

Comment:
"equitable" does not appear to make sense here.
consider removing, or alternatively replace with "clean" or somesuch

Reply:
I disagree because this is a word used for UI/UX design, it means "accessibility" if I can recall

Reply:
mmmmmmmmm
that makes sense, then